packages/components/src/tabs/stories/best-practices.mdx

```diff
@@ -0,0 +1,92 @@
+import { Meta } from '@storybook/blocks';
+
+import * as TabsStories from './index.story';
+
+<Meta of={ TabsStories } name="Best Practices" />
+
+# Tabs
+
+## Usage
+
+### Uncontrolled Mode
+
+Tabs can be used in an uncontrolled mode, where the component manages its own state. In this mode, the `defaultTabId` prop can be used to set the initially selected tab. If this prop is not set, the first tab will be selected by default. In addition, in most cases where the currently active tab becomes disabled or otherwise unavailable, uncontrolled mode will automatically fall back to selecting the first available tab.
+
+```jsx
+import { Tabs } from '@wordpress/components';
+
+const onSelect = ( tabName ) => {
+  console.log( 'Selecting tab', tabName );
```

Comment:
Are we intentionally using 2 spaces instead of a single tab here?

Reply:
Me personally, yes. I find it very hard to read those tabbed code snippets in places where the tab width is fixed at 8 spaces 😅

Reply:
I find it quite the opposite FWIW. I'm used to 4-space tabs and what I'm seeing here feels so packed 😱

Reply:
Ok my brain exaggerated, everywhere I perceived as tab width 8 ("super wide!") was just 4 😂

- Block Editor Handbook: 4
- Storybook: 4
- GitHub Markdown Preview: 4

The only objective reasoning I can think of to prefer 2 spaces is because the automatic code snippets in Storybook use that (and is not configurable).

I'm fine with unifying back to tabs. That is what our codebase mandates, after all 🤷

Reply:
😅 Thanks, yes, unifying back to tabs makes sense.

Reply:
Done in e7fa2d1cb13b46e96734b8ec11a7091bf8d15d13